app/views/organisations/show.html.haml

```diff
@@ -18,12 +18,12 @@
         %dt
           = t("organisation.type")
         %dd
-          = @organisation.organisation_type
+          = t("organisation.organisation_type.#{@organisation.organisation_type}")
         %dt
           = t("organisation.language_code")
         %dd
-          = @organisation.language_code
+          = t("organisation.language_code.#{@organisation.language_code.downcase}")
```

Comment:
I think downcasing is a very good idea to make sure this works reliably. It prompts to we think we should probably have some test coverage for it too.

From there I think it might make sense to have this set on a global getter for an organisation record rather than in the views, so they don't have to remember _every_ single time. This then avoids us having to test this low level edge case with features, and can use faster unit tests for it too!

We could add a spec to the existing model but then I think that data logic be the same as view logic, they should be decoupled so that how we process data isn't affected by the presentational logic. It should be able to be capitalised in our database, with no affect on the views.

This would lead me to create the  first presenter object for an organisation, with 2 small unit tests.

```
# app/presenters
class OrganisationPresenter < SimpleDelegator
  def language_code
    super.downcase
  end

  def default_currency
    super.downcase
  end
end

app/organisation_controller.rb
@organisation = OrganisationPresenter.new(organisation)
```